helm/harbor-scanner-trivy/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: harbor-scanner-trivy
-version: 0.1.5
-appVersion: "0.2.3"
+version: 0.1.6
```

Comment:
Not really sure what this version should be. Any ideas?

Reply:
> Not really sure what this version should be. Any ideas?

As long as we bump it up to make helm upgrade happy we're fine.